### ISSUE TITLE
ArConnect test failing

### ISSUE THREAD

**Issue by rosmcmahon** [ArweaveTeam/arweave-js#126]
test for external wallet is failing as permissions not given before signing.
`it("should create and sign transactions using external implicit wallet",`

Need to:

1. get the signing permission from the user/tester
2. then run the failing test 
3. make this run in a github workflow. (headless chrome module with chrome and firefox?)

**Comment by hlolli** [comment]
is this still failing?

**Comment by rosmcmahon** [comment]
yes. i've been waiting on a solution from arweave.app instead of getting into a whole puppeteer testing arrangement with arconnect

**Comment by rosmcmahon** [comment]
should probably just remove this test. also, should not be importing types from "arconnect" dep, should be declared in arweave-js.
